platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -353,7 +352,6 @@ public PlatformComponentBuilder buildComponentBuilder() {
         }
 
         // FUTURE WORK: remove after static use of cryptography is abolished
```

Comment:
The cryptography holder was removed but the comment about removing the cryptography holder was not.

Reply:
Comment is still relevant. There is also static reference to `MerkleCryptography` we still set here. I consider this to be part of getting rid of static cryptography.

Reply:
Comment removed. I also found I had to move the static instantiation of merkle cryptography to the context builder (issue exposed by failing JRS test).